lib/ui/src/modules/ui/components/theme.js

```diff
@@ -1,7 +1,7 @@
 export const baseFonts = {
   fontFamily: `
-    -apple-system, ".SFNSText-Regular", "San Francisco", "Roboto",
-    "Segoe UI", "Helvetica Neue", "Lucida Grande", sans-serif
+    -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu",
+    "Cantarell", "Fira Sans", "Droid Sans", "Arial", sans-serif
```

Comment:
Yes it is. I think it's to conform to the system's original styles and Arial is an unfamiliar and clunky looking font to Mac users.